esy/System.ml

```diff
@@ -22,6 +22,11 @@ let host =
     | "Cygwin" -> Cygwin
     | _ -> Unknown
 
+let envSep = 
```

Comment:
I wasn't sure if this was the best spot for it, but wanted a place to call out the per-platform environment separator. LMK if you think this belongs somewhere else, @andreypopp .